_includes/live-demos/custom-shortcut-2/index.html

```diff
@@ -0,0 +1,3 @@
+<textarea id="custom-shortcut-2">
+  <p>Use the shortcut <code>Cmd+Option+U</code> on Mac or <code>Ctrl+Alt+U</code> if your on Windows or press the plus button to insert a username here: </p>
```

Comment:
your -> you are. Also commas maybe?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1651/commits/1b33c5835a4e1b0494839449271cb103f5074764